src/components/marker.ts

```diff
@@ -32,6 +33,11 @@ export type MarkerProps<OptionsT, MarkerT extends MarkerInstance> = OptionsT & {
   children?: React.ReactNode;
 };
 
+// Adapted from https://github.com/mapbox/mapbox-gl-js/blob/v1.13.0/src/ui/popup.js
+function getClassList(className: string) {
+  return new Set(className ? className.trim().split(/\s+/) : []);
+}
```

Comment:
Copied this over from Popup component - can add `src/utils/get-class-list.ts` and use that util in both Marker and Popup component. Let me know what you think is best.